scripts/release/helpers.py

```diff
@@ -0,0 +1,9 @@
+import subprocess
+import shlex
+
+
+def run(cmd):
+    parts = shlex.split(cmd)
```

Comment:
I'm confused why this is needed and not `*args` - shell quoting is nasty enough passing args though seems tidier to me (but there may well be reasons here that I'm missing)

Reply:
I just like being able to write:

```
run("git push --tags")
```

as opposed to

```
run(["git", "push", "--tags"])
```

Of course, we can always drop back to the underlying `subprocess.run` method for more complicated cases, but for the simpler cases I think this is more readable.